GitUI/CommandsDialogs/SettingsDialog/Pages/ToolbarSettingsPage.Designer.cs

```diff
@@ -0,0 +1,110 @@
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Component Designer generated code
+
+        /// <summary> 
+        /// Required method for Designer support - do not modify 
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            this.components = new System.ComponentModel.Container();
+            this.tableLayoutPanelForToolbar = new System.Windows.Forms.TableLayoutPanel();
+            this.label2 = new System.Windows.Forms.Label();
+            this.cbBranchOrderingCriteria = new System.Windows.Forms.ComboBox();
+            this.tooltip = new System.Windows.Forms.ToolTip(this.components);
+            this.tableLayoutPanelForToolbar.SuspendLayout();
+            this.SuspendLayout();
+            // 
+            // tableLayoutPanelForToolbar
+            // 
+            this.tableLayoutPanelForToolbar.AutoSize = true;
+            this.tableLayoutPanelForToolbar.ColumnCount = 2;
+            this.tableLayoutPanelForToolbar.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.tableLayoutPanelForToolbar.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.tableLayoutPanelForToolbar.Controls.Add(this.label2, 0, 0);
+            this.tableLayoutPanelForToolbar.Controls.Add(this.cbBranchOrderingCriteria, 1, 0);
+            this.tableLayoutPanelForToolbar.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.tableLayoutPanelForToolbar.Location = new System.Drawing.Point(0, 0);
+            this.tableLayoutPanelForToolbar.Name = "tableLayoutPanelForToolbar";
+            this.tableLayoutPanelForToolbar.RowCount = 9;
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.Size = new System.Drawing.Size(1333, 684);
+            this.tableLayoutPanelForToolbar.TabIndex = 1;
```

Comment:
this layout may not work well under Mono, I will check it in coming days

Reply:
If that's the case, can you explain why? I've never worked with Mono.

Reply:
Mono has troubles rendering flexible layouts, layouts we take for granted on Windows platform don't render well under Mono.
There a number of reported issues https://github.com/gitextensions/gitextensions/issues?utf8=%E2%9C%93&q=is%3Aissue%20%20label%3Alinux%2Fmono%20label%3AUX%20